### DISCUSSION TITLE
give name to backup

### DISCUSSION THREAD

**Discussion by punkyard** [nextcloud/all-in-one#2713] [Ideas] [answered]
hi,
mastercontainer is a great tool, to update and backup.
But in a list of backups with dates, it's difficult to remember who is who.
Could it be possible to name the backup?
thanks for the great work 🤞 still testing it

**Comment by szaimen** [comment, marked as answer]
Hi, I fear this idea is out of scope for AIO as it only offers a minor advantage in my opinion but would make the logic much more complex and harder to maintain and thus more unreliable. 

I would recommend to note down the timestamps externally in order remember which was which.
